libnavigation-core/src/test/java/com/mapbox/navigation/core/trip/session/MapboxTripSessionTest.kt

```diff
@@ -58,13 +58,15 @@ class MapboxTripSessionTest {
     private val tripStatus: TripStatus = mockk(relaxUnitFun = true)
 
     private val routeProgress: RouteProgress = mockk()
+    private val navigatorPollingDelay = 1500L
 
     @Before
     fun setUp() {
         tripSession = MapboxTripSession(
```

Comment:
don't you think that it's strange create instance of `MapboxTripSession` for each test but don't use it, and create new one inside test fun?  🤔

Reply:
It's used in a half of the tests at least. I'm not sure that it's the best solution to move all creations inside the tests